app/services/common_platform/api/court_application_record_representation_order.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module CommonPlatform
+  module Api
+    class CourtApplicationRecordRepresentationOrder < ApplicationService
+      def initialize(case_defendant_offence:,
+                     subject_id:,
+                     offence_id:,
+                     status_code:,
+                     application_reference:,
+                     status_date:,
+                     effective_start_date:,
+                     defence_organisation:,
+                     effective_end_date: nil,
+                     connection: CommonPlatform::Connection.call)
+        @case_defendant_offence = case_defendant_offence
+        @offence_id = offence_id
+        @status_code = status_code
+        @application_reference = application_reference.to_s
+        @status_date = status_date
+        @effective_start_date = effective_start_date
+        @effective_end_date = effective_end_date
+        @defence_organisation = defence_organisation
+        @url = "prosecutionCases/representationOrder"\
+                "/applications/#{case_defendant_offence.prosecution_case_id}"\
```

Comment:
The other advantage of my previous comment is that this line could instead be:

```
"/applications/#{application_defendant_offence.court_application_id}"\
```

Which I think makes a little more sense if you don't know the weird way that the DB is structured.

Reply:
Updated this too thanks.